src/domains/chart/sagas.ts

```diff
@@ -136,9 +137,14 @@ function* fetchDataSaga({ payload }: Action<FetchDataPayload>) {
       group,
       gtime,
       agent_options: options.split("|"),
-      aggregation: {
-        method: aggrMethod,
+      aggregations: [groupBy === "node" && {
```

Comment:
Not sure I understand this. Can you please explain what you have agreed for that with @lvrach ?
In case of `nodes` grouping I see this will be a list of aggregations? 
```
     [{
       method: "sum",
        groupBy: ["chart", "node"],
      },
      {
        method: aggrMethod,
        groupBy: [groupBy],
      }]
```
Can you explain this a bit?